### DISCUSSION TITLE
Wiim minis become unavailable if linked in the Wiim app

### DISCUSSION THREAD

**Discussion by JimmyMPage** [music-assistant/support#3112] [Q&A] [answered]
Hello,

Tonight I noticed two of our Wiim mini players (set up as Airplay devices) were showing unavailable in Home Assistant. These players are provided my Music Assistant, so I checked there and found them missing as well. I asked my son (who uses the Wiim app to play his music) if he saw any strange behavior tonight. He said he had been playing the two missing players as a linked pair through the Wiim app and that everything worked fine. This led me to investigate a bit and it seems if the players are linked to play through the Wiim app, they become unavailable in Home Assistant. I am not sure if this behavior is due to their configuration as Airplay devices in MA or if it's some other issue. I understand this may be an issue that cannot be resolved because of some background functionality between the devices and the native Wiim app.

My son likes the app because it's super easy for him to use, but I have encouraged him to pull up Home Assistant and use Music Assistant to link his desired players and play from there...we will see how that goes. It WOULD be ideal if the players would somehow recover after being used as linked pair (or more) from the Wiim app.

Hopefully that all makes sense. I'm out of town on work until Sunday, but I can still do some troubleshooting from here if required.

**Comment by OzGav** [comment]
What version of MA are you using? How have the Wiims been added…AirPlay?

**Reply by JimmyMPage** [reply]
I am running MA 2.3.2. The wiims are added as airplay devices in MA. 

![image](https://github.com/user-attachments/assets/2c518c37-1e9f-4dbc-8648-9601353c8976)

They are the "players" in the above list. When they are linked together in the Wiim app (on a phone or other device), they will disappear from this list and show as "unavailable" entities in Home Assistant. It seems to require a reboot of each offending device to get them to show up again, though I haven't had a chance to sit and watch for a long time (>1 hr) to see if they eventually come back after being unlinked from the app.

**Reply by OzGav** [reply]
Ok will see what @marcelveldt says about this…

**Reply by marcelveldt** [reply, marked as answer]
This is a device/wiim issue, not MA. Apparently they remove them as airplay target when they become passive (linked to another player), which kind of makes sense because they can no longer be targeted directly. But whe you unlink them, they should start broadcasting their (raop) mdns messages again so the MA Airplay provider knows they're available again.

Anyways, this is probably a bit of useless discussion anyways as MA will very soon have native linkplay support.
